solr/cross-dc-manager/src/test/org/apache/solr/crossdc/manager/SimpleSolrIntegrationTest.java

```diff
@@ -46,21 +45,21 @@ public static void ensureWorkingMockito() {
 
   @BeforeClass
   public static void beforeSimpleSolrIntegrationTest() throws Exception {
-
+    System.setProperty("solr.crossdc.bootstrapServers", "doesnotmatter:9092");
+    System.setProperty("solr.crossdc.topicName", "doesnotmatter");
     cluster1 =
         configureCluster(2)
-            .addConfig("conf", getFile("configs/cloud-minimal/conf").toPath())
+            //           .addConfig("conf", getFile("configs/cloud-minimal/conf").toPath())
```

Comment:
I don't recall, it's been a while, but if the test is passing, I assume I needed to do it and then forgot to remove it. That would be a little surprising, but if the test is passing, I don't see why else it would have been commented out.